cypress/e2e/urlHandling.cy.ts

```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
 
       cy.title().should('eq', 'Test | Playroom');
     });
+
+    it('editor hidden', () => {
+      cy.visit(
+        'http://localhost:9000/#?code=N4IgpgJglgLg9gJwBJQhMA7EAuGCCuYAvkA'
+      );
+
+      cy.get('textarea').should('not.be.focused');
+
+      // Todo - write a test that checks the CodeMirror element is not visible
+      /*
+      The CodeMirror element is not visible, but it is in the DOM
+      This test fails because the element doesn't meet Cypress's requirements for being hidden
+      https://docs.cypress.io/guides/core-concepts/interacting-with-elements#An-element-is-considered-hidden-if
+      */
+      // cy.get('.CodeMirror-code').should('be.hidden');
+    });
```

Comment:
This is proving very difficult to test. I don't think it is worth holding up the release unless there is an easy fix